From e4aff292fcc622735530daacb56c85bb3da0dc59 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 1 Aug 2018 11:46:54 +0800
Subject: [PATCH 102/153] dt-bindings: clock: sun8i-de2: add H6 DE3 CCU
 compatible string

The Allwinner H6 SoC comes with "Allwinner DE3", which is a modified
version of DE2. From the perspective of clock, a ROT clock is added, but
the other clock keeps the same.

Add compatible string for the DE3 CCU. The ROT clock slice number will
be added when introducing the H6 DE3 CCU support.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/clock/sun8i-de2.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/clock/sun8i-de2.txt b/Documentation/devicetree/bindings/clock/sun8i-de2.txt
index e94582e8b8a9..e12214e19827 100644
--- a/Documentation/devicetree/bindings/clock/sun8i-de2.txt
+++ b/Documentation/devicetree/bindings/clock/sun8i-de2.txt
@@ -8,6 +8,7 @@ Required properties :
 		- "allwinner,sun8i-v3s-de2-clk"
 		- "allwinner,sun50i-a64-de2-clk"
 		- "allwinner,sun50i-h5-de2-clk"
+		- "allwinner,sun50i-h6-de3-clk"
 
 - reg: Must contain the registers base address and length
 - clocks: phandle to the clocks feeding the display engine subsystem.
-- 
2.18.1

